src/filelock/_unix.py

```diff
@@ -45,6 +48,7 @@ def _acquire(self) -> None:
             try:
                 fcntl.flock(fd, fcntl.LOCK_EX | fcntl.LOCK_NB)
             except OSError as exception:
+                _LOGGER.debug("Failed to acquire lock: %s", exception)
```

Comment:
It would be better to use `exc_info=True` to include full details of the exception (type, message, traceback) rather than just the message alone.

Like this:
```suggestion
                _LOGGER.debug("Failed to acquire lock", exc_info=True)
```

https://docs.python.org/3/library/logging.html#logging.Logger.debug